app/routes.py

```diff
@@ -0,0 +1,152 @@
+import logging
+
+from flask import abort, render_template, jsonify, make_response, request
+
+from werkzeug.exceptions import HTTPException
+
+from shapely.geometry import Point
+
+from app import app, georaster_utils
+from app.helpers import make_error_msg
+from app.helpers.route import prefix_route
+from app.helpers.height_helpers import get_height
+from app.helpers.profile_helpers import get_profile
+
+from app.helpers.validation import srs_guesser, bboxes
+from app.helpers.validation.height import validate_lon_lat, validate_sr
+import app.helpers.validation.profile as profile_arg_validation
+
+# add route prefix
+from app.statistics.statistics import load_json, prepare_data
+
+app.route = prefix_route(app.route, '/rest/services/')
+
+logger = logging.getLogger(__name__)
+
+
+@app.errorhandler(Exception)
+def handle_exception(e):
+    # pass through HTTP errors
+    if isinstance(e, HTTPException):
+        return e
+    logger.error(str(e))
+    return make_error_msg(500, "Internal server error, please consult logs")
+
+
+@app.route('/checker', methods=['GET'])
+def check():
+    return make_response(jsonify({'success': True, 'message': 'OK'}))
+
+
+@app.route('/height')
+def height_route():
+    if 'easting' in request.args:
+        lon = request.args.get('easting')
+    else:
+        lon = request.args.get('lon')
+    if 'northing' in request.args:
+        lat = request.args.get('northing')
+    else:
+        lat = request.args.get('lat')
+    (lon, lat) = validate_lon_lat(lon, lat)
+
+    if 'sr' in request.args:
+        sr = int(request.args.get('sr'))
+    else:
+        point = Point(lon, lat)
+        sr = srs_guesser(point)
+        if sr is None:
+            abort(make_error_msg(400, "No 'sr' given and cannot be guessed from 'geom'"))
+    sr = validate_sr(sr)
+
+    if lon < bboxes[sr][0] or lon > bboxes[sr][2] or lat < bboxes[sr][1] or lat > bboxes[sr][3]:
+        abort(make_error_msg(400, "Query is out of bounds"))
+    alt = get_height(sr, lon, lat, georaster_utils)
+    if alt is None:
+        abort(
+            make_error_msg(
+                400, 'Requested coordinate ({},{}) out of bounds in sr {}'.format(lon, lat, sr)
+            )
+        )
+    return {'height': str(alt)}
```

Comment:
As it is not only used by us, but by many customers, I don't think it's a good idea to change this return structure without extensive pre-warning for our customers.

Reply:
True here we need to defined if we want to release it under a new API major version or not. If we release to a new major API version then I think it is ok, espacialy because it is just extending the answer. If we don't release it under a new major version I would leave as it is.